change/@fluentui-react-components-79050853-fe98-49ee-994b-b9dca0dd661a.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=29194&commitId=2880047a562d2ee595753d562550b5789222ab17&env=prod&clientType=FLUENTUI)


  <details><summary><b>Tree</b> 8 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Tree.default open tree.chromium.png|871| Changed|
|Tree.default open tree - RTL.chromium.png|876| Changed|
|Tree.default open tree - Dark Mode.chromium.png|791| Changed|
|Tree.default open tree - High Contrast.chromium.png|1174| Changed|
|Tree.persona - Dark Mode.chromium.png|492| Changed|
|Tree.persona - High Contrast.chromium.png|901| Changed|
|Tree.persona.chromium.png|653| Changed|
|Tree.persona - RTL.chromium.png|653| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>